Plots/XY/NCL_xy_2.py

```diff
@@ -0,0 +1,64 @@
+"""
+NCL_xy_2.py
+===============
+This script illustrates the following concepts:
+   - Drawing an XY plot with multiple curves
+   - Changing the line color for multiple curves in an XY plot
+   - Changing the line thickness for multiple curves in an XY plot
+   - Drawing XY plot curves with both lines and markers
+   - Changing the default markers in an XY plot
+   - Making all curves in an XY plot solid.
+See https://www.ncl.ucar.edu/Applications/xy.shtml for further information.
+"""
+
+###############################################################################
+# Import packages
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import matplotlib.ticker as tic
+from cartopy.mpl.ticker import LongitudeFormatter, LatitudeFormatter
+
+###############################################################################
+# Open a netCDF data file using xarray default engine and load the data into xarrays
+ds = xr.open_dataset("../../data/netcdf_files/uv300.nc")
+u = ds.U[:]
+
+###############################################################################
+# Generate plot data from netCDF data
+# xx = u.values[0,:,82]
+xx = u[0,:,:].sel(lon=[82], method='nearest')
+# yy = u.values[0,:,-69]
+yy = u[0,:,:].sel(lon=[-69], method='nearest')
+lat = u['lat'].values
+
+###############################################################################
+# Multiple line plot
+fig = plt.figure()
+ax = plt.axes()
+
+# Hard-code tic values. This assumes data are global
+ax.set_xticks(np.linspace(-90, 90, 7))
+ax.set_yticks(np.linspace(-20, 50, 8))
+
+# Hard-code x-axis tick labels
+ax.set_xticklabels(['90S', '60S', '30S', '0', '30N', '60N', '90N'])
```

Comment:
maybe use cartopy's `LatitudeFormatter` here?

Reply:
Agree that `Cartopy LatitudeFormatter` could be used here, but it would cost loading a projection. Since this is only a XY-plot, it is considered unnecessary and just hard-coded. FYI: This issue was discussed in the first-day's concluding reviews.

Reply:
:+1: